pkg/configfilearg/parser.go

```diff
@@ -101,14 +103,36 @@ func (p *Parser) findStart(args []string) ([]string, []string, bool) {
 		return []string{}, args, true
 	}
 
+	afterIndex := make(map[string]int)
+	re, err := regexp.Compile(`(.+):(\d)+`)
```

Comment:
You want to capture all the digits in this group, not allow the group to match multiple times
```suggestion
	re, err := regexp.Compile(`(.+):(\d+)`)
```

Reply:
Thanks! Fixed